Tests/TemplatesTests/TestsHelper.swift

```diff
@@ -116,8 +116,9 @@ class Fixtures {
     return string(for: name, subDirectory: "templates/\(sub.rawValue.lowercased())")
```

Comment:
All of these changes can be removed, they are not needed. Use the existing variations mechanism.